lib/fastlane/plugin/wpmreleasetoolkit/actions/android/android_send_app_size_metrics.rb

```diff
@@ -3,13 +3,23 @@
 module Fastlane
   module Actions
     class AndroidSendAppSizeMetricsAction < Action
+      # Keys used by the metrics payload
+      AAB_FILE_SIZE_KEY = 'AAB File Size'.freeze                     # value from `File.size` of the `.aab`
+      UNIVERSAL_APK_FILE_SIZE_KEY = 'Universal APK File Size'.freeze # value from `File.size` of the Universal `.apk`
+      UNIVERSAL_APK_SPLIT_NAME = 'Universal'.freeze                  # pseudo-name of the split representing the Universal `.apk`
```

Comment:
Just curious: Why is at "pseudo-name"?

Reply:
Because for all the other splits, the name (i.e. the value used for the `split:` metadata) is provided by `bundletool` (it's the basename of the `.apk` produced by it in the `<tmpdir>/splits/*.apk`); while for the universal APK it's just a name of our choosing and not some official name provided by the Android tooling, so it could be any name we want it to be.

In fact if we wanted to be pedantic, the Universal APK should probably not be considered a "split" per se — it's not the name of a CPU architecture after all. So it's a bit like if we sais for iOS that the Universal `.ipa` was a "thinned version for the device named 'Universal': `pseudo-name` because not really the name of a split/CPU-arch but one we made up to represent/mean "multiple archs in a single FAT file".